spec/config_spec.rb

```diff
@@ -151,6 +151,46 @@
     expect(Settings.size).to eq(2)
   end
 
+  describe "load_and_set_settings in the Rails context" do
```

Comment:
Awesome tests!

They seem to be failing on CI though. There might be something going on with test execution order as well since locally sometimes I get 3 failures, and other times just one. I'm able to reproduce the failures locally on ruby 3.1.4

```sh
bundle exec appraisal rails-6.1 rspec
```

Reply:
Hi! 

I achieved stable running of tests locally. `Config::Options` test cleared ENV. So `rails runner` executed with empty ENV and this led to an error starting rails (Gem not found).